### PR TITLE
feat(apps/interface): add cakerise and cakedrop icons

### PR DESCRIPTION
**Pull request by asaadam**
add cakerise and cakedrop icons

### CODE REVIEW

File: apps/interface/public/icons/CAKERISE-light.svg [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<svg width="48" height="49" viewBox="0 0 48 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pyk) [inline apps/interface/public/icons/CAKERISE-light.svg:1]: @asaadam set height and view box to `48`

File: apps/interface/public/icons/CAKERISE-dark.svg [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<svg width="52" height="54" viewBox="0 0 52 54" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pyk) [inline apps/interface/public/icons/CAKERISE-dark.svg:1]: @asaadam set height and viewbox to 48

File: apps/interface/public/icons/CAKEDROP-light.svg [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<svg width="48" height="49" viewBox="0 0 48 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pyk) [inline apps/interface/public/icons/CAKEDROP-light.svg:1]: set height and width viewbox to 48

File: apps/interface/public/icons/CAKEDROP-dark.svg [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<svg width="52" height="54" viewBox="0 0 52 54" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (pyk) [inline apps/interface/public/icons/CAKEDROP-dark.svg:1]: Set width, height and viewbox to 48

**Review by pyk — CHANGES_REQUESTED** [review]

**Comment by asaadam** [comment]
@pyk wait ser there's some request change to from @ayikfour, I will update the icon now

**Review by pyk — APPROVED** [review]
Thanks @asaadam 💯

**Pull request merged**